components/NavigationDrawer/examples/BottomDrawerExpandFullscreenExample.swift

```diff
@@ -16,7 +16,6 @@ import UIKit
 import MaterialComponents.MaterialBottomAppBar
 import MaterialComponents.MaterialColorScheme
 import MaterialComponents.MaterialNavigationDrawer
-import MaterialComponents.MaterialNavigationDrawer_ColorThemer
```

Comment:
Can the ColorThemer target be removed from the podspec/BUILD file?

Reply:
I've been doing that part when I delete the targets. I'm sticking to a rough plan of 1) remove usage 2) deprecate 3) remove from library and delete.

Reply:
Next time, please remove the deps when they are no longer needed rather than when the target is deleted.